guides/v2.1/cloud/reference/docker-config.md

```diff
@@ -0,0 +1,69 @@
+---
+group: cloud
+title: Configure Docker
+version: 2.1
+github_link: cloud/reference/docker-config.md
+functional_areas:
+  - Cloud
+  - Setup
+  - Configuration
+---
+
+{{site.data.var.ece}} provides a Docker environment option for those who use their local environment for develop and testing, or for automation tasks. There are three, essential parts to launching the Magento Cloud Docker environment: a Magento 2 template, Docker Compose, and Magento Cloud Tools.
+
+The [Magento Cloud Docker repository](https://github.com/magento/magento-cloud-docker){:target="\_blank"} contains build information for the following [Docker hub](https://hub.docker.com/r/magento/){:target="\_blank"} images, including the following:
+
+-  PHP: magento/magento-cloud-docker-php
+    -  PHP-CLI - version 7 and later
+    -  PHP-FPM - version 7 and later
+-  NGINX:  magento/magento-cloud-docker-nginx
+
+**Prerequisites**—Select your Magento 2 template.
+
+1.  Download a template from the [Magento Cloud repository](https://github.com/magento/magento-cloud){:target="\_blank"}.
+1.  Add your credentials to `auth.json` file.
+1.  Update the template dependencies.
+
+    ```bash
+    composer install
+    ```
+
+#### To launch the Cloud Docker environment:
+The Cloud tools provide a command to generate the Docker Compose configuration.
+
+1.  In your local environment, start the Docker configuration generator.
+
+    ```bash
+    vendor/bin/ece-tools docker:build
```

Comment:
After this step you need to copy default files:

```
cp docker/config.env.dist docker/config.env
cp docker/global.env.dist docker/global.env
```

Reply:
remove: ", including the following"

earlier in the sentence it already says: ...for the following...